src/main.ts

```diff
@@ -68,7 +68,11 @@ const launchMooLite = () => {
         app.mount(
             (() => {
                 const app = document.createElement("div");
-                document.body.append(app);
+                const root = document.getElementById("root");
+                console.log(document.body.classList);
+                root?.append(app);
```

Comment:
Instead of having root possibly be null, can you add something like

```ts
if (root == null) {
  throw new Error("Could not append MooLite to the element with id 'root' as it does not exist");
}
```

Reply:
Done